Source/WebCore/bindings/js/JSDOMConvertStrings.cpp

```diff
@@ -118,4 +121,94 @@ AtomString valueToUSVAtomString(JSGlobalObject& lexicalGlobalObject, JSValue val
     return replaceUnpairedSurrogatesWithReplacementCharacter(WTFMove(string));
 }
 
+String valueToTrustedHTMLString(JSGlobalObject& lexicalGlobalObject, JSValue value, const String& sink, bool nullToEmpty)
+{
+    UNUSED_PARAM(sink);
```

Comment:
Style-wise we prefer to comment out, or omit, the `sink` name in the arguments list above. `UNUSED_PARAM` should only be used when we can’t do that for some reason.

Reply:
Ah okay good to know. Will change that.